packages/x-components/src/components/modals/__tests__/base-id-modal-close.spec.ts

```diff
@@ -18,7 +18,7 @@ function renderBaseIdModalClose({
   // Making bus not repeat subjects
   jest.spyOn(bus, 'createEmitter' as any).mockImplementation(dummyCreateEmitter.bind(bus) as any);
 
-  const [, localVue] = installNewXPlugin();
+  const [, localVue] = installNewXPlugin(undefined, undefined, bus);
```

Comment:
You can avoid it in every test sheet, changing every `bus.` use by `XPlugin.bus`. In this way, all test sheets will use the same bus, the dummy one. 

There are two types of "listeners" used:
```TS
XPlugin.bus.on('UserClickedCloseModal').subscribe(listener);
instead of
bus.on('UserClickedCloseModal').subscribe(listener);
```

```TS
emitSpy: jest.spyOn(XPlugin.bus, 'emit'),
instead of 
emitSpy: jest.spyOn(bus, 'emit'),
```

Moreover, if you do that change, you can also remove everything related with async events 🥳 
```TS
// Making bus not repeat subjects
  jest.spyOn(bus, 'createEmitter' as any).mockImplementation(dummyCreateEmitter.bind(bus) as any);

jest.runAllTimers();

beforeAll(() => {
  jest.useFakeTimers();
});
```